GSA_Adapter/CRUD/ReadResults.cs

```diff
@@ -255,6 +255,29 @@ private bool CheckAnalysisCaseResultsExists(int caseId, string caseDescription)
         }
 
         /***************************************************/
+
+        private int TryExtractMode(string loadcase)
+        {
+            if (loadcase.Contains("A"))
+                loadcase = loadcase.Replace("A", "");
```

Comment:
If I assume that this makes sense in GSA land and won't lead to strange loadcase names for unsuspecting users.

Reply:
Only used for extracting the mode number. Modal cases are called like "A5" etc, and the string sent in here will be of that type (when mode is available). The API call extracting the mode number only wants an int though, so this is simply cleaning up the string for the APIs sake. The string should remain untouched for what the users see.